### PR TITLE
Completed Bubble_Sort Project

### PR DESCRIPTION
**Pull request by angelacuahutle**
In this project, We implemented the bubble sort algorithm using Ruby. We understood each part of this method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+- Ruby
+
+## Live Demo
+
+[Live Demo]
+
+## Getting Started
+clone the repository $ git clone https://github.com/angelacuahutle/Ruby_bubble_sort.git
+go to the project directory $ cd bubble_sort
+Run the command $ ruby bubble_sort.rb
```
Comment (BrianSammit) [inline README.md]: You can improve this using bullet points.
Example:
- Open terminal 
- $ git clone where you want the project.
- ...

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+go to the project directory $ cd bubble_sort
+Run the command $ ruby bubble_sort.rb 
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+
+### Install
```
Comment (BrianSammit) [inline README.md:24]: You are not using this, please delete it or use it to improve your Readme.

**Review by BrianSammit — CHANGES_REQUESTED** [review]
REQUIRED CHANGES :recycle: 
------------
Hello team :wave:  Excellent job you did so far :muscle: 
However, there are somethings that you forget to do. :thinking: 
------------

- [x] You forgot to do the #bubble_sort_by method. you can find the specifications [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421588/).

Screenshot from 2021-01-22 22-03-27

--------------------------
Make these changes and request for another review. :smile: 
HAPPY CODING!!

**Comment by angelacuahutle** [comment]
Made changes as suggested:

- We build the #bubble_sort_by method.
- Fixed Readme file. 
- Setup and installed linters

File: README.md [hunk 3]
```diff
@@ -1,2 +1,68 @@
...
+In this project, We implemented the bubble sort algorithm. 
+
+## Built With
+
+- Ruby
+
+## Live Demo
+
+[Live Demo]
+
```
Comment (GzimAsani) [inline README.md:11]: [Optional] - There is no live demo for this project so kindly remove this

File: README.md [hunk 4]
```diff
@@ -1,2 +1,68 @@
...
+-Go to the project directory $ cd bubble_sort
+-Run the command $ ruby bubble_sort.rb 
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+
+### Install
```
Comment (GzimAsani) [inline README.md:24]: Also remove these since you are not using them

**Review by GzimAsani — APPROVED** [review]
# Approved ✅

## You have done a great  job making this and improving the code based on the previous review 👏🧨💪!

### You have met all the requirements of the project. Go ahead and merge it! 🚀 !

_Please check the optional changes for the README file which suggestions will make it look more professional!_
<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:
Happy coding 👨‍💻

**Comment by angelacuahutle** [comment]
Project approved

**Pull request merged**